man/crypttab.xml

```diff
@@ -260,6 +260,29 @@
         <xi:include href="version-info.xml" xpointer="v243"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>link-volume-key-to-keyring=</option></term>
```

Comment:
that's quite a mouthful... 

maybe call this just "link-volume-key"?

I mean, "keyring" alone is kinda too short anyway I think, i.e. at least to me it's not entirely obvious that this would mean the *kernel* keyring. But if we'd say "…-kernel-keyring" in the option then it would get even longer... hence maybe drop that parse altogether?

I think without the suffix it's not really worse: "link-volume-key" indicates the volume key shall be linked into something, there are not too many areas it could be placed in?

yes, i am arguing here to wards giving this a less precise name, because the proposed name is too vague in my eyes. If you say "this makes no sense", then I would understand you ;-)

naming is hard.